### PR TITLE
Implement support for working-directories

### PR DESCRIPTION
**Pull request by bastelfreak**
I tested this with the beaker.yml in my private control repository and it works.

### CODE REVIEW

File: .github/workflows/basic.yml [hunk 1]
```diff
@@ -14,9 +14,17 @@ on:
         default: 45
         required: false
         type: number
+      working-directory:
+        description: The working directory where all jobs should be executed. useful if you've a module in a subdirectory (like a monorepo or a controlrepository)
```
Comment (ekohl) [inline .github/workflows/basic.yml]: I generally prefer the documentation to be impersonal
```suggestion
        description: The working directory where all jobs should be executed. Used for modules in subdirectories like a monorepo or a control repository.
```

**Review by ekohl — COMMENTED** [review]
Interesting idea. What I did for foreman-infra was to actually implement things rather manual at the level above:
https://github.com/theforeman/foreman-infra/blob/master/puppet/Rakefile

This way we didn't need to implement it for every module and you get an aggregated result. I still think that should be made easier though because control repos are common.

**Comment by bastelfreak** [comment]
I considered an implementation at the top level, but this approach allows me to work only wirh files in the module directory which in turn makes it easier to migrate the module into its own git repo.

**Comment by ekohl** [comment]
Just to be clear, I'm not against this but I also think repo-level multi module testing should be easier than it is today.

File: .github/workflows/beaker.yml [hunk 2]
```diff
@@ -19,9 +19,17 @@ on:
         default: 45
         required: false
         type: number
+      working-directory:
+        description: The working directory where all jobs should be executed. useful if you've a module in a subdirectory (like a monorepo or a controlrepository)
```
Comment (ekohl) [inline .github/workflows/beaker.yml]: I sort of implied to update this and the other instance as well to keep them in sync.

Comment (bastelfreak) [reply .github/workflows/beaker.yml]: yes I want to update them, but Ibak currently only on my phone :D will do it later

Comment (bastelfreak) [reply .github/workflows/beaker.yml]: should be fine now.

**Review by ekohl — APPROVED** [review]

**Pull request merged**